src/test/java/com/cloudant/tests/CloudantClientTestBase.java

```diff
@@ -0,0 +1,82 @@
+package com.cloudant.tests;
+
+import com.cloudant.client.api.CloudantClient;
+
+import org.junit.After;
+import org.junit.Before;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Created by Rhys Short on 21/05/15.
+ */
+public abstract class CloudantClientTestBase {
```

Comment:
Changed in b1f5025 and 89459e8 to make the base class a helper with a `getClient()` method